ecc/blocks/event-info-component/event-info-component.js

```diff
@@ -94,7 +94,7 @@ export default function init(el) {
         generateToolTip(r);
         break;
       case 1:
-        await decorateTextfield(r, { id: 'info-field-event-title' });
+        await decorateTextfield(r, { id: 'info-field-event-title' }, 'This event title is already taken');
```

Comment:
Probably we can get the message from word document? and can it be - "This event title is already taken for a given event series, date and venue combination"